qgreenland/util/provenance.py

```diff
@@ -1,7 +1,24 @@
+from pathlib import Path
+
+from qgreenland.models.config.layer import Layer
 from qgreenland.models.config.step import AnyStep
 
 
 def steps_to_provenance_text(steps: list[AnyStep]) -> str:
     steps_as_text = [step.provenance for step in steps]
 
     return '\n\n'.join(steps_as_text)
+
+
+def write_provenance_file(*, layer_cfg: Layer, filepath: Path) -> None:
+    """Write layer provenance to a text file."""
+    # TODO: default message for layers with no processing steps?
+    txt_to_write = ''
```

Comment:
Should layers with no steps have an empty provenance file? Maybe we can have a message "NO PROCESSING STEPS TAKEN BY QGREENLAND"?

Reply:
Maybe we should find a way to include a representation of how the data was fetched in the provenance file, and for those with no steps, leave it at that? "Data fetched from ___" and that's it?

Reply:
Not a bad idea. I've updated this TODO and created a separate task for this.